pkg/kube/client.go

```diff
@@ -87,6 +89,7 @@ import (
 	"istio.io/istio/pkg/sleep"
 	"istio.io/istio/pkg/test/util/yml"
 	"istio.io/istio/pkg/version"
+	"istio.io/istio/tools/bug-report/pkg/common"
```

Comment:
? I donot think we should allow import this package

Reply:
Thanks, fixed.